src/Http/Authentication.Abstractions/src/IAuthenticationHandler.cs

```diff
@@ -16,7 +16,7 @@ public interface IAuthenticationHandler
         /// </summary>
         /// <param name="scheme">The <see cref="AuthenticationScheme"/> scheme.</param>
         /// <param name="context">The <see cref="HttpContext"/> context.</param>
-        /// <returns></returns>
+        /// <returns>A task that represents the asynchronous operation.</returns>
```

Comment:
Can we remove `returns` for Task returning APIs? This canned comment doesn't add any value. See how elsewhere when the method returns a `Task<T>` you only describe the T?

Reply:
Let me see if I can get someone from the docs team to weigh in on this. It looks fairly reasonable but is inconsistent with the runtime docs: 

* https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.authentication.iauthenticationhandler.initializeasync?view=aspnetcore-3.1#returns

* https://docs.microsoft.com/en-us/dotnet/api/system.io.memorystream.flushasync?view=net-5.0#returns

Reply:
I checked with @BillWagner on this. The framework APIs display it because of compliance requirements. I'm awaiting a response from @mimisasouvanh on what displays on the API ref page if we remove `<returns>`.